### ISSUE TITLE
Convert all px to rem as part of the build

### ISSUE THREAD

**Issue by wesruv** [patternfly/patternfly-elements#970]
It's my (somewhat contentious) opinion that `rem` units are better for a11y than px, and I also feel that it's a robot's job to do the conversion for us.

There's a PostCSS plugin that converts all `px` to `rem` that I'd love to see added to the build for SCSS > CSS. It should be a drop in improvement that doesn't cause any bugs (I've done this on multiple projects) but we'd want to test.

If anyone wants I can bore everyone with why I think `rem` and `em`s make for better a11y.

**Comment by starryeyez024** [comment]
If HTML sets the root differently than you expect, this can be problematic
https://www-admin.corp.stage.redhat.com/en/jobs/pre-employment

**Comment by wesruv** [comment]
Waaaait a second... since this is REM inside of the shadow DOM, doesn't that mean REM is whatever the component styles say, and the parent DOM doesn't matter?

**Comment by starryeyez024** [comment]
Negative, the rem unit still looks up to the HTML parent and inherits that base value. Even if the component were not using a rem unit, the font-size property would still be inherited at the shadow DOM level, as long as there were no other interfering properties.

**Comment by wesruv** [comment]
Nuts.

**Comment by wesruv** [comment]
Well I think it's worth closing this then, seems like it's enough that it could blow up on us.

**Comment by castastrophe** [comment]
For your consideration: `font-size: clamp(16px, 1rem, 20px);`

This lets us set limits to prevent unreasonably small or large results. It defaults to 1rem but not if it is smaller or larger than the defined limits. Great browser support.

**Comment by wesruv** [comment]
IMO I'm more worried about small results. If someone wants to jack up their font size to 30 because they like that size I'm cool with it. If everything on the page is em/rem or relative units it'll be l like they zoomed in a lot.
